subsys/bluetooth/controller/Kconfig

```
@@ -584,5 +584,13 @@ config BT_CTLR_SDC_CS_COUNT
 	help
 	  Set the number of concurrent connections that can support Channel Sounding procedure.
 
+config BT_CTLR_SDC_LE_POWER_CLASS_1
+	bool "Device supports transmitting at LE Power Class 1 level"
+	default n
```

We could consider setting the default value based upon `BT_CTLR_TX_PWR_ANTENNA`:
```suggestion
	default y if BT_CTLR_TX_PWR_ANTENNA >= 10
```
That ensures that devices using a high static power level will set the bit correctly

---

Good idea!